test/spec/PptTestData.scala

```diff
@@ -122,102 +121,119 @@ trait PptTestData extends RegistrationBuilder with MockAuthAction {
 
   protected val testBusinessVerificationPassStatus = "PASS"
 
-  protected val incorporationRegistrationDetails: IncorporationRegistrationDetails =
-    IncorporationRegistrationDetails(registeredBusinessPartnerId =
-                                       Some(safeNumber),
-                                     registrationStatus = "REGISTERED"
+  protected val registrationDetails: RegistrationDetails =
+    RegistrationDetails(identifiersMatch = true,
+                        verificationStatus = Some("PASS"),
+                        registrationStatus = "REGISTERED",
+                        registeredBusinessPartnerId = Some(safeNumber)
+    )
+
+  protected val unregisteredRegistrationDetails: RegistrationDetails =
+    RegistrationDetails(identifiersMatch = true,
+                        verificationStatus = Some("UNCHALLENGED"),
+                        registrationStatus = "REGISTRATION_NOT_CALLED",
+                        registeredBusinessPartnerId = None
     )
 
-  protected val unregisteredIncorporationRegistrationDetails: IncorporationRegistrationDetails =
-    IncorporationRegistrationDetails(registeredBusinessPartnerId = None,
-                                     registrationStatus = "REGISTRATION_NOT_CALLED"
+  protected val verificationFailedRegistrationDetails: RegistrationDetails =
+    RegistrationDetails(identifiersMatch = true,
+                        verificationStatus = Some("FAIL"),
+                        registrationStatus = "REGISTRATION_NOT_CALLED",
+                        registeredBusinessPartnerId = None
+    )
+
+  protected val identifiersUnmatchedRegistrationDetails: RegistrationDetails =
```

Comment:
I'm not very familiar with all of the different Business Verification responses but these details are identical to `unregisteredRegistrationDetails`, is this correct?

Reply:
The `identifiersMatch` flag is different.

Reply:
Doh! Apologies and thanks for pointing out the difference :sweat_smile: :+1: